src/modules/pattern/pattern.service.js

```diff
@@ -0,0 +1,30 @@
+const Pattern = require('./pattern.model');
```

Comment:
Please, move all newly added logic for patterns to another merge request to merge it in development as quickly as possible.